lib/routes/account.js

```diff
@@ -959,10 +980,13 @@ module.exports = function (
         }
 
         function createResponse() {
-          return {
-             email: sessionToken.email,
-             verified: sessionToken.emailVerified
-           }
+          return db.sessionTokenVerified(sessionToken.id)
+          .then(function(token) {
+            return {
+              email: token.email,
+              verified: !!(token.emailVerified && token.tokenVerified)
```

Comment:
>  telling a little white lie about the verification status of the account :-(

Agree, it is sad, though backwards compatible. We are changing the semantics from `account` to `account and session`.